tutorials/manage-container-registry-images/index.mdx

```diff
@@ -52,69 +52,72 @@ This tutorial will show you how to periodically remove older images with a speci
-        req = urllib.request.Request(url, headers=headers)
-        with urllib.request.urlopen(req) as response:
-            res = response.read()
-            img_list=eval(res)["images"]
-        i=i+1
-        if not img_list:
-            browse=False
-        else:
-            images_list.extend(img_list)
-    tags_to_delete=[]
-    
-    # Get all tags by image to delete
-    
-    for image in images_list:
-        url=f"https://api.scaleway.com/registry/v1/regions/{region}/images/{image['id']}/tags?page_size=100&order_by=created_at_desc"
-        req = urllib.request.Request(url, headers=headers)
-        with urllib.request.urlopen(req) as response:
-            res = response.read()
-            tags=eval(res)["tags"]
-        i=0
-        for tag in tags:
-            if i<tags_to_keep:
-                i=i+1
-            else:
-                tags_to_delete.append(tag)
-    
-    #Delete tags and return their status
-    
-    msg=[]
-    for tag in tags_to_delete:
-        tag_id=tag['id']
-        url_delete=f"https://api.scaleway.com/registry/v1/regions/{region}/tags/{tag_id}"
-        req = urllib.request.Request(url_delete, headers=headers,method="DELETE")
-        with urllib.request.urlopen(req) as response:
-            res = response.read()
-            msg.append(json.loads(res.decode("utf-8")))
-    
-    return {
-        "body": {
-            "message":msg
-        },
-        "statusCode": 200,
-    }
+      images_list=[]
+      browse=True
+      i=1
+
+      # Get the list of all images on all pages  
+      while browse:
+          url=f"https://api.scaleway.com/registry/v1/regions/{region}/images?page_size=100&order_by=created_at_asc&page="+str(i)
+          req = urllib.request.Request(url, headers=headers)
+          with urllib.request.urlopen(req) as response:
+              res = response.read()
+              img_list=eval(res)["images"]
+          i=i+1
+          if not img_list:
+              browse=False
+          else:
+              images_list.extend(img_list)
+      tags_to_delete=[]
+
+      # Get all tags by image to delete
+
+      for image in images_list:
+          url=f"https://api.scaleway.com/registry/v1/regions/{region}/images/{image['id']}/tags?page_size=100&order_by=created_at_desc"
+          req = urllib.request.Request(url, headers=headers)
+          with urllib.request.urlopen(req) as response:
+              res = response.read()
+              tags=eval(res)["tags"]
+          i=0
+          for tag in tags:
+              if i<tags_to_keep:
+                  i=i+1
+              else:
+                  tags_to_delete.append(tag)
+
+      #Delete tags and return their status
+
+      msg=[]
+      for tag in tags_to_delete:
+          tag_id=tag['id']
+          url_delete=f"https://api.scaleway.com/registry/v1/regions/{region}/tags/{tag_id}?force=true"
```

Comment:
Using `force=true` can be dangerous, as it may delete the most recent tags if they share the same manifest digest with any older tags listed in `tags_to_delete`.

Reply:
You may have several tags sharing the same manifest digest.

Reply:
Sorry for the delay. I just removed the `force=true` parameter.

Ready to merge.